core/node/da_dispatcher/src/da_dispatcher.rs

```diff
@@ -14,11 +20,14 @@ use zksync_types::L1BatchNumber;
 
 use crate::metrics::METRICS;
 
+const NO_FIRST_BATCH: i64 = -1;
```

Comment:
I don't like having this constant here. We should rethink a bit how we do this in the future.

Reply:
It has been moved to the corresponding config file.